sickbeard/server/web/config/subtitles.py

```diff
@@ -0,0 +1,96 @@
+                      subtitles_download_in_pp=None, subtitles_keep_only_wanted=None):
+        """
+        Save Subtitle Search related settings
+        """
+        results = []
+
+        config.change_SUBTITLES_FINDER_FREQUENCY(subtitles_finder_frequency)
+        config.change_USE_SUBTITLES(use_subtitles)
+
+        sickbeard.SUBTITLES_LANGUAGES = [code.strip() for code in subtitles_languages.split(',') if code.strip() in subtitles.subtitle_code_filter()] if subtitles_languages else []
+        sickbeard.SUBTITLES_DIR = subtitles_dir
+        sickbeard.SUBTITLES_PERFECT_MATCH = config.checkbox_to_value(subtitles_perfect_match)
+        sickbeard.SUBTITLES_HISTORY = config.checkbox_to_value(subtitles_history)
+        sickbeard.EMBEDDED_SUBTITLES_ALL = config.checkbox_to_value(embedded_subtitles_all)
+        sickbeard.SUBTITLES_HEARING_IMPAIRED = config.checkbox_to_value(subtitles_hearing_impaired)
+        sickbeard.SUBTITLES_MULTI = 1 if len(sickbeard.SUBTITLES_LANGUAGES) > 1 else config.checkbox_to_value(subtitles_multi)
+        sickbeard.SUBTITLES_DOWNLOAD_IN_PP = config.checkbox_to_value(subtitles_download_in_pp)
+        sickbeard.SUBTITLES_KEEP_ONLY_WANTED = config.checkbox_to_value(subtitles_keep_only_wanted)
+        sickbeard.SUBTITLES_EXTRA_SCRIPTS = [x.strip() for x in subtitles_extra_scripts.split('|') if x.strip()]
+        sickbeard.SUBTITLES_PRE_SCRIPTS = [x.strip() for x in subtitles_pre_scripts.split('|') if x.strip()]
+
+        # Subtitles services
+        services_str_list = service_order.split()
+        subtitles_services_list = []
+        subtitles_services_enabled = []
+        for curServiceStr in services_str_list:
+            curService, curEnabled = curServiceStr.split(':')
+            subtitles_services_list.append(curService)
+            subtitles_services_enabled.append(int(curEnabled))
+
+        sickbeard.SUBTITLES_SERVICES_LIST = subtitles_services_list
+        sickbeard.SUBTITLES_SERVICES_ENABLED = subtitles_services_enabled
+
+        sickbeard.ADDIC7ED_USER = addic7ed_user or ''
+        sickbeard.ADDIC7ED_PASS = addic7ed_pass or ''
+        sickbeard.ITASA_USER = itasa_user or ''
+        sickbeard.ITASA_PASS = itasa_pass or ''
+        sickbeard.LEGENDASTV_USER = legendastv_user or ''
+        sickbeard.LEGENDASTV_PASS = legendastv_pass or ''
+        sickbeard.OPENSUBTITLES_USER = opensubtitles_user or ''
+        sickbeard.OPENSUBTITLES_PASS = opensubtitles_pass or ''
+
+        sickbeard.save_config()
+        # Reset provider pool so next time we use the newest settings
+        subtitles.get_provider_pool.invalidate()
+
+        if results:
+            for x in results:
+                logger.log(x, logger.ERROR)
+            ui.notifications.error('Error(s) Saving Configuration',
+                                   '<br>\n'.join(results))
+        else:
+            ui.notifications.message('Configuration Saved', ek(os.path.join, sickbeard.CONFIG_FILE))
```

Comment:
Just out of curiosity, wasn't it that we didn't need ek() anymore, because of the future unicode_literals?